File: C-Sharp-part-2/03.Methods/04. Appearance count.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

class appearanceCount
{
    static int CountOfNumberX(int[] numbers, int X)
    {
        int count = 0;
        for (int i = 0; i < numbers.Length; i++)
        {
            if (numbers[i] == X)
            {
                count++;
            }
        }
        return count;
    }

    static void Main()
    {
        int arr = int.Parse(Console.ReadLine());
        int[] numbers = new int[arr];
        string[] input = Console.ReadLine().Split(' ');
        for (int i = 0; i < input.Length; i++)
        {
            numbers[i] = int.Parse(input[i]);
        }
        int X = int.Parse(Console.ReadLine());

        Console.WriteLine(CountOfNumberX(numbers, X));
    }
}

File: JavaScript-Fundamentals/05.Functions/04. Appearance count.js
function count(input) {
    "use strict";

    var arrLength = input[0];
    var arr = input[1].split(' ');

    var count = 0;
    for (var i = 0; i < arr.length; i += 1) {
        if (arr[i] === input[2]) {
            count += 1;
        }
    }
    console.log(count);
}

//count(['8', '28 6 21 6 -7856 73 73 -56', '73']);